### DISCUSSION TITLE
sentry transactions not found

### DISCUSSION THREAD

**Discussion by maazrehman1** [getsentry/support#75] [Support] [answered]
### Is there an existing issue for this?

- [X] I have checked for existing issues https://github.com/getsentry/sentry-javascript/issues
- [X] I have reviewed the documentation https://docs.sentry.io/
- [X] I am using the latest SDK release https://github.com/getsentry/sentry-javascript/releases

### How do you use Sentry?

Sentry Saas (sentry.io)

### Which package are you using?

@sentry/nextjs

### SDK Version

7.8.0

### Framework Version

React 14.15

### Steps to Reproduce

<img width="967" alt="Screenshot 2022-08-29 at 9 23 35 PM" src="https://user-images.githubusercontent.com/91028677/187249243-df0a5002-2759-493f-a747-33990b08b587.png">
<img width="468" alt="Screenshot 2022-08-29 at 9 23 47 PM" src="https://user-images.githubusercontent.com/91028677/187249272-5e2218f0-baa1-4369-b9ae-aa02c8747443.png">

![Uploading Screenshot 2022-08-29 at 9.32.37 PM.png…]()

### Expected Result

i implemented sentry and was able to monitor all the events on my project, around in 9 days i say last record of my project having 13k transactions. it should provide me proper event report along with errors and performance monitoring in the future too but it broke ...

### Actual Result

 suddenly all of them were gone and there was no new transactions created. the records are showing 0 transactions record, although i still get errors with proper source maps etc but im unable to track my transactions anymore. i can still see the amount of transactions but can not get their details.

**Comment by AbhiPrasad** [comment, marked as answer]
Hey thanks for writing in! Could you check to see if you have [exhausted your transaction quota here](https://sentry.io/orgredirect/organizations/:orgslug/stats/)? If you've run out of quota, we will no longer ingest transaction events. To help manage quota, see docs accordingly: https://help.sentry.io/account/billing/i-ve-exhausted-my-quota-what-now/
